css/main.css

```diff
@@ -325,3 +325,63 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+/* Footer */
+
+.footerText {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+}
+
+.footerForm {
+  padding: 10%;
+  padding-bottom: 20vh;
+  text-align: center;
+}
+
+.footerLegend {
+  margin-bottom: 30px;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 44px;
+  text-align: center;
+}
+
+.formInput {
+  width: 75%;
+  font-weight: 200;
+  font-size: 1.2rem;
+  line-height: 24px;
+  border: none;
+  padding: 10px 25px;
+  margin: 10px;
+  color: #979493;
+  border-bottom: 1px solid #dbd8d7;
+}
+
+fieldset {
+  border: none;
+}
+
+.formTextArea {
+  font-size: 1.2rem;
+  width: 75%;
+  height: 114px;
+  background-color: #f5eeec;
+  font-weight: normal;
+  line-height: 24px;
+  border: none;
+  padding: 10px 25px;
+  margin: 10px;
+}
+
+.footerBtn {
```

Comment:
- [ ] OPTIONAL
According to the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) you are supposed to Implement the button interactions (enable, hover, etc.) would you fix that? Having button interactions on your website helps improve the User Experience. Sorry, the previous reviewer didn't notice this.